src/sentry/utils/http.py

```diff
@@ -19,6 +20,12 @@
 ParsedUriMatch = namedtuple('ParsedUriMatch', ['scheme', 'domain', 'path'])
 
 
+class FilterTypes(object):
```

Comment:
Yeah, I was questioning that as well. Wondering if @mattrobenolt or @tkaemming have opinions on where these should live?